### PR TITLE
Check for corrupted data

### PR DESCRIPTION
**Pull request by kiote**
# Why

  sometimes HPR service returns corrupted on incomplete html page and we need to
  handle this situation gently in the gem

# How
  
  throw error in case of corrupted data

# Tests

With autotests only

### CODE REVIEW

File: lib/hpr.rb [hunk 1]
```diff
@@ -6,4 +6,5 @@
 module Hpr
   class MissingMedicalAuthorizationError < ArgumentError; end
   class InvalidHprNumberError < MissingMedicalAuthorizationError; end
+  class CantGetPersonError < ArgumentError; end
```
Comment (kiote) [inline lib/hpr.rb]: Not sure about this name, but, naturally, we can't get a person data from HPR!

File: lib/hpr/scraper.rb [hunk 2]
```diff
@@ -99,6 +99,8 @@ def approval_boxes
 
     def person_header
       @person_header ||= page.at_css(".person-header")
+      raise Hpr::CantGetPersonError, "Hpr number: #{hpr_number}" unless @person_header
```
Comment (kiote) [inline lib/hpr/scraper.rb]: here actually the root of problems with "at_css" for nil. We have no header == no reason to look further

File: spec/fixtures/corrupted_data/6125735.html [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+        <div class="header-right">
+            <a href="http://www.sak.no/" title="Statens Autorisasjonskontor for helsepersonell (SAK)">
+                <img src="./6172733_files/logo-sak.png" alt="Statens Autorisasjonskontor for helsepersonell (SAK)">
+            </a>
+        </div>
+
+    </div>
+
+            </header>
```
Comment (kiote) [inline spec/fixtures/corrupted_data/6125735.html]: I've cut this file let's say randomly :)

File: spec/spec_helper.rb [hunk 4]
```diff
@@ -4,17 +4,7 @@
 require_relative "../lib/hpr"
 
 def stub_hpr_request(number, fixture)
-  if number.empty?
-    number = 'empty'
-    return stub_request(:post, "https://register.helsedirektoratet.no/Hpr/Hpr/Lookup").
-        with(:body => {"Number"=>""},
-             :headers => {'Accept'=>'*/*; q=0.5, application/xml', 'Accept-Encoding'=>'gzip, deflate', \
```
Comment (kiote) [inline spec/spec_helper.rb]: Not sure if we need this sophisticated stubbing, with headers, etc?
Seems like it's enough to have just what I leave.

File: lib/hpr.rb (hunk 1 above)
Comment (rogerkk) [inline lib/hpr.rb]: Dunno if these are any better, but how about `PersonReadError`, `PersonFetchError`, `PersonLookupError` or perhaps referring to exactly what goes wrong here and call it something like `PersonHeaderNotFoundError` or`ScrapingError`?

File: spec/spec_helper.rb (hunk 4 above)
Comment (rogerkk) [inline spec/spec_helper.rb]: If you say so! :)

File: lib/hpr.rb (hunk 1 above)
Comment (kiote) [inline lib/hpr.rb]: I was thinking about `ScrapingError` as well, so will rename!

**Pull request merged**